### DISCUSSION TITLE
Hello! I hope you're having a good afternoon 😊. I have a small question that I would like to clarify.

### DISCUSSION THREAD

**Discussion by FrankieD777** [komikku-app/komikku#247] [Q&A]
How can I prevent duplicate chapters from appearing in mangas? I think I modified something by mistake.
![Screenshot_20240810-121158](https://github.com/user-attachments/assets/bcf18d30-b8f4-4614-9963-443dcca23327)

**Comment by kana-shii** [comment]
Try refreshing the entry, this should remove the duplicate chapters. If it doesn't, I don't think there's much we can do. Although it could be an extension problem.
For further questions and for a follow up on this question, please talk to us through discord so we can easily help you.
https://discord.com/invite/85jB7V5AJR

**Reply by FrankieD777** [reply]
Hello, good afternoon. I hope you are well.

When I mention updating the entry, I mean swiping from top to bottom to refresh the page and display the recent chapters. If you've already tried this and it's still the same, it might be an issue with the extension.

If you have more questions or want to follow up on this issue, don't hesitate to contact us through Discord so we can help you more effectively: [https://discord.com/invite/85jB7V5AJR](https://discord.com/invite/85jB7V5AJR).

Best regards!

**Comment by FrankieD777** [comment]
It remains the same. Even after updating the entire library.
![Screenshot_20240810-131935](https://github.com/user-attachments/assets/a0327052-db76-4688-920c-08ef2620bff5)
![Screenshot_20240810-132538](https://github.com/user-attachments/assets/606512ea-24db-4577-80fc-5f0f6b0588e1)

**Reply by kana-shii** [reply]
It must be an extension problem then. I recommend creating an issue (probably both on Keiyoushi's repo and in Komikku's repo just to be sure)
For now, the only thing you could possibly do is bookmark the extra chapters and filter the entry by chapters without bookmarks or just wait for a fix.
And again, for a follow up, please talk to us through discord instead.

**Comment by cuong-tran** [comment]
If this issue has recently appeared after you perform a backup restoration, you can try this:
1. Remove the entry from the library (un-heart it)
2. Go to `Settings -> Advanced -> Clear database`. From there, tick the source which has the issue entry. Hit `Delete`, *un-tick the `Keep entries with read chapters`* then hit OK.
3. Go to source, add the entry again.

**Reply by FrankieD777** [reply]
If this problem has appeared recently after performing a backup restoration, you can try the following:

1. Remove the entry from the library (unmark it)
2. Go to `Settings -> Advanced -> Clear database`. From there, mark the source that contains the problematic entry. Press `Delete`, _uncheck the box and `Keep entries with read chapters`_, then press OK.
3. Go to the source and add the entry again.

It didn't work ;-;.

**Comment by Trived7** [comment]
Remove the unwanted sources and keep only what you wanted, you can find it in your pic there above the filter option(3 lines) and there scanlator and choose the wanted sources you want your chapters from.

**Reply by cuong-tran** [reply]
This is not the case of the original people asking. He got duplicated chapters caused by a backup restoration, same chapter - not from different scanlators. This issue is common for all Mihon's fork.
I got a fix comming up to avoid this.

**Reply by Trived7** [reply]
My bad and thanks for telling me about that.
